src/ui/public/vislib/lib/layout/layout.js

```diff
@@ -50,13 +54,39 @@ export default function LayoutFactory(Private) {
      * @returns {*} Creates the visualization layout
      */
     createLayout(arr) {
-      const self = this;
-
-      return _.each(arr, function (obj) {
-        self.layout(obj);
+      return _.each(arr, (obj) => {
+        this.layout(obj);
       });
     };
 
+    updateCategoryAxisSize() {
```

Comment:
wrong sizing of category axis (it was either too long or too short) and it gets rid of that magic number (which worked just in case your value axis was 43 pixels wide/tall :)